torch/csrc/jit/fuser/executor.cpp

```diff
@@ -262,17 +252,22 @@ void launchFusion(
   }
 
   // Adds (flattened) output arguments
+  outputs.resize(fusion.outputDesc().size());
```

Comment:
Can we push_back inside the loop instead? That would be even faster, since we wouldn’t have to construct empty tensor pointers.

Reply:
Good point!

Reply:
Also, can keep reserve as done previously